### PR TITLE
Add support for node 18

### PR DESCRIPTION
**Pull request by JavanPoirier**
As per #105, adds support for node 18 to engines and GH workflows.

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
         os: [windows-2019, ubuntu-latest, macos-latest]
         # list only the earliest and latest node versions supported
         # this makes PR builds more efficient
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
```
Comment (byt3pool) [inline .github/workflows/pr.yml:19]: If I read the comment correctly, this should be:

```suggestion
        node-version: [14, 18]
```

File: .github/workflows/merge.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-2019, macos-latest, ubuntu-latest]
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
```
Comment (byt3pool) [inline .github/workflows/merge.yml:22]: I think this should be:

```suggestion
        node-version: [14, 18]
```

(the earliest and latest node versions supported)

File: .github/workflows/pr.yml (hunk 1 above)
Comment (JavanPoirier) [reply .github/workflows/pr.yml]: Updated, thanks. Yea, I clearly missed that.

**Comment by jshor** [comment]
Thanks for submitting this PR. Node v18 and v20 are supported as of v4.0.0.

**Pull request closed without merging**